src/mono/wasm/runtime/loader/config.ts

```diff
@@ -201,6 +201,7 @@ export function normalizeConfig() {
     }
 
     runtimeHelpers.diagnosticTracing = loaderHelpers.diagnosticTracing = !!config.diagnosticTracing;
+    runtimeHelpers.aot = config.aot || false;
```

Comment:
I disagree with this, it should not be in the boot config. It should be added into `dotnet.es6.lib.js` as link time variable and it would become a JS code constant.